.github/workflows/main.yml

```diff
@@ -26,3 +26,28 @@ jobs:
 
       - name: Run shellcheck on all scripts
         run: make shellcheck
+        if: false # We will enable this once all the scripts are passing
+
+      - name: Create kind cluster
+        uses: helm/kind-action@v1.0.0
+        if: steps.lint.outputs.changed == 'true'
+  
+      - uses: azure/setup-kubectl@v1
```

Comment:
Thoughs on using these available actions? I'm putting more trust in `azure/<whatever>` than in `random/<whatever>`